l10n_mx_edi_addenda_audi/views/l10n_mx_edi_addenda_audi_view.xml

```diff
@@ -23,7 +23,7 @@
                 <t t-foreach="record.invoice_line_ids" t-as="move_line">
                     <S:Parte
                         t-att-precioUnitario="move_line.price_unit"
-                        t-att-numeroParte="move_line.line_external_code"
+                        t-att-numeroParte="move_line.audi_line_product_ref"
```

Comment:
I would go with something like this:

```suggestion
                        t-att-numeroParte="move_line.product_id.audi_ref"
```

No need for a new field on the account move line